### PR TITLE
Update Intel DAOS community examples to use google-cloud-daos v0.3.0

### PR DESCRIPTION
**Pull request by mark-olson**
The DAOS Terraform modules in the daos-stack/google-cloud-daos repository were updated to work with DAOS v2.2.0. The updates were tagged v0.3.0.

As a result the community/examples/intel/daos-*.yaml blueprints were updated to use the v0.3.0 terraform modules in the daos-stack/google-cloud-daos repository.

Signed-off-by: Mark Olson <115657904+mark-olson@users.noreply.github.com>

### Submission Checklist

* [x] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [ ] Are all tests passing? (`make tests`)
* [ ] Have you written unit tests to cover this change?
* [ ] Is unit test coverage still above 80%?
* [x] Have you updated all applicable documentation?
* [x] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

File: community/examples/intel/README.md [hunk 1]
```diff
@@ -3,35 +3,34 @@
...
-    - [Delete the DAOS infrastructure when not in use](#delete-the-daos-infrastructure-when-not-in-use)
-  - [DAOS Server with Slurm cluster](#daos-server-with-slurm-cluster)
-    - [Initial Setup for the DAOS/Slurm cluster](#initial-setup-for-the-daosslurm-cluster)
-    - [Deploy the DAOS/Slurm Cluster](#deploy-the-daosslurm-cluster)
-    - [Connect to the DAOS/Slurm Cluster login node](#connect-to-the-daosslurm-cluster-login-node)
-    - [Create and Mount a DAOS Container](#create-and-mount-a-daos-container)
-    - [Run a Job that uses the DAOS Container](#run-a-job-that-uses-the-daos-container)
-    - [Unmount the Container](#unmount-the-container)
-    - [Delete the DAOS/Slurm Cluster infrastructure when not in use](#delete-the-daosslurm-cluster-infrastructure-when-not-in-use)
+- [Intel-Optimized Slurm Cluster](#intel-optimized-slurm-cluster)
```
Comment (tpdownes) [inline community/examples/intel/README.md:6]: @mark-olson I believe that you accidentally removed the top-level TOC item here.

When I run

```
md_toc github -l 4 community/examples/intel/README.md
```

I see that there are no changes from the develop branch. I propose the following:

1. You revert the changes to the TOC
2. You or I update the comment on line 3 to include the command I show above. If you do it, please include it in this PR. If you would rather I do it, a Toolkit team member will submit a separate PR.

File: community/examples/intel/README.md [hunk 2]
```diff
@@ -336,7 +335,7 @@ For the purpose of this demo create the container without specifying ACLs. The c
 daos cont create pool1 \
   --label cont1 \
   --type POSIX \
-  --properties rf:1
+  --properties rf:0
```
Comment (tpdownes) [inline community/examples/intel/README.md:338]: Is this intentional? I don't see a mention in commit and I lack the relevant expertise.

File: community/examples/intel/README.md [hunk 3]
```diff
@@ -378,6 +377,7 @@ See the [File System](https://docs.daos.io/v2.0/user/filesystem/) section of the
 The container will need to by unmounted before you log out.  If this is not done it can leave open file handles and prevent the container from being mounted when you log in again.
 
 ```bash
+popd
```
Comment (tpdownes) [inline community/examples/intel/README.md:380]: Is this intentional? I don't see a mention in commit and I lack the relevant expertise.

**Review by tpdownes — CHANGES_REQUESTED** [review]
@mark-olson 

As always, thanks for the commit and for resolving your CLA signature a couple weeks ago. I have relatively minor concerns.

1. Revert accidental modification of the TOC for the Intel README.
2. Address probably minor questions.

I have not yet tested the solution, but that might be best for @cboneti who will return from travel before I do.

File: community/examples/intel/README.md (hunk 1 above)
Comment (tpdownes) [reply community/examples/intel/README.md:6]: If you lack easy access to https://pypi.org/project/md-toc/ in your working environment, please let me know.

**Comment by mark-olson** [comment]
This PR is outdated and had some issues.  Closing it. I will submit a new one.

**Pull request closed without merging**